spec/index.md

```diff
@@ -639,8 +647,60 @@ Calls must stay in `replied` or `rejected` long enough for polling users to catc
 
 When asking the IC about the state or call of a request, the user uses the request id (see [Request ids](#request-id)) to read the request status (see [Request status](#state-tree-request-status)) from the state tree (see [Request: Read state](#http-read-state)).
 
+#### Synchronous canister calling {#http-sync-call-overview}
+
+A synchronous update call, also known as a "call and await", is a type of update call where the replica will attempt to respond to the HTTPS request with a certificate of the call status. If the returned certificate indicates that the update call is in a terminal state (`replied`, `rejected`, or `done`), then the user __does not need to poll__ (using [`read_state`](#http-read-state) requests) to determine the result of the call. A terminal state means the call has completed its execution.
+
+The synchronous call endpoint is useful for users as it removes the networking overhead of polling the IC to determine the status of their call.
+
+The replica will maintain the HTTPS connection for the request and will respond once the call status transitions to a terminal state. 
+
+If an implementation specific timeout for the request is reached while the replica waits for the terminal state, then the replica will reply with an empty body and a 202 HTTP status code. In such cases, the user should use [`read_state`](#http-read-state) to determine the status of the call.
+
 ### Request: Call {#http-call}
 
+In order to call a canister, the user makes a POST request to `/api/v3/canister/<effective_canister_id>/call`. The request body consists of an authentication envelope with a `content` map with the following fields:
+
+-   `request_type` (`text`): Always `call`
+
+-   `sender`, `nonce`, `ingress_expiry`: See [Authentication](#authentication)
+
+-   `canister_id` (`blob`): The principal of the canister to call.
+
+-   `method_name` (`text`): Name of the canister method to call.
+
+-   `arg` (`blob`): Argument to pass to the canister method.
+
+The HTTP response to this request can have the following forms:
+
+-   200 HTTP status with a non-empty body. This status is returned if the canister call completed or was rejected within an implementation-specific timeout.
```

Comment:
```suggestion
-   200 HTTP status with a non-empty body. This status is returned if the canister call completed.
```

Reply:
The call can be rejected, and we still reply with `200`. Hence the `status` fields below.

Reply:
> The call can be rejected, and we still reply with 200. Hence the status fields below.

What do you mean?

Reply:
I mean the ingress filter can fail, and we never submit the ingress message to consensus+p2p, we reply with a `200` status code.

https://github.com/dfinity/interface-spec/pull/265/files/d6edfe5ab5d10c2b08e0d6267fe88c224841232e#diff-b9a833d6114611a080aee4cd57618feb45aa044bcc11105d44b155b5bbde00ebR684-R693

Reply:
> I mean the ingress filter can fail, and we never submit the ingress message to consensus+p2p, we reply with a 200 status code.

This is covered by "the canister call completed" since the call completed in this case (with an error).